src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/metadata/ProjectSampleMetadataController.java

```diff
@@ -132,14 +132,22 @@ public SampleMetadataStorage createProjectSampleMetadata(HttpSession session, @P
 				Iterator<Cell> cellIterator = row.cellIterator();
 				while (cellIterator.hasNext()) {
 					Cell cell = cellIterator.next();
+
 					int columnIndex = cell.getColumnIndex();
 					if (columnIndex < headers.size()) {
 						String header = headers.get(columnIndex);
 
 						if (!Strings.isNullOrEmpty(header)) {
 							// Need to ignore empty headers.
-							cell.setCellType(CellType.STRING);
-							rowMap.put(header, cell.getStringCellValue());
+							if(cell.getCellTypeEnum().equals(CellType.NUMERIC)) {
+								DataFormatter formatter = new DataFormatter();
+								String value = formatter.formatCellValue(cell);
```

Comment:
This seems to work for me, glad it was that straightforwad.  Even rounding seems to happen the same as expected.  Can you add a comment about why it's being added?